integrations/Cargo.toml

```diff
@@ -29,18 +29,18 @@ cosmwasm-std = { version = "1.2" }
 cosmwasm-schema = { version = "1.2" }
 cw-storage-plus = "1.1"
 cw2 = { version = "1.0" }
-cw20 = { git = "https://github.com/AbstractSDK/cw-plus.git", version = "1.0.1", tag = "v1.1.1" }
+cw20 = { git = "https://github.com/AbstractSDK/cw-plus.git", version = "1.0.1", rev = "03274f3" }
```

Comment:
Let's create a new tag for these as well

Reply:
Done